README.md

```diff
@@ -1,27 +1,69 @@
-# Sirius
+<h1 align="center">
+  <br>
+  <a href="https://CS-SI.github.io/SIRIUS/html/Sirius.html"><img src="https://github.com/dumasl/SIRIUS/blob/enhanced_readme/doc/img/sirius-logo.svg" alt="Sirius by CS" width=60%></a>
```

Comment:
Image URL is incorrect
Replace with `https://github.com/CS-SI/SIRIUS/blob/master/doc/img/sirius-logo.svg`